torch/csrc/jit/codegen/cuda/executor.h

```diff
@@ -202,6 +230,12 @@ class TORCH_CUDA_CU_API FusionExecutor : public NonCopyable {
     return &compile_time_info_cache_;
   }
 
+  //! ignoring aliased outputs, pushing scalar int 0 as a place holder
+  KernelArgumentHolder evaluateOutputSizes(
```

Comment:
I think this function could still use a better comment. I know what you mean but it might not be clear to other people.